normandy/recipes/models.py

```diff
@@ -85,6 +87,13 @@ def arguments(self):
     def arguments(self, value):
         self.arguments_json = json.dumps(value)
 
+    @property
+    def current_approval_request(self):
+        try:
+            return self.approval_requests.get(active=True)
```

Comment:
To be more specific:

1. There's nothing stopping someone from making multiple approval requests currently, so it's technically possible.
2. I suspect that multiple users will want to make changes to a recipe at the same time (such as the main heartbeat recipe, which hosts multiple surveys) and I'm not sure if we're handling that anywhere, either blocking it or allowing it. I haven't yet looked at the previous peer approval PR though, so I may be missing something there. I'll take a look.